src/main/com/yetanalytics/datasim/util/errors.clj

```diff
@@ -0,0 +1,18 @@
+(ns com.yetanalytics.datasim.util.errors
+  (:require [clojure.spec.alpha :as s]))
+
+(defn explain-to-map-coll
+  "Convert spec error data for personae, alignments, and parameters
+   into a map coll acceptable to the Datasim UI."
+  [spec-kw spec-ed]
+  (mapv
+   (fn [problem]
+     (let [epath (into [spec-kw] (:in problem))
+           estr  (-> problem
+                     (assoc ::s/problems [problem])
+                     s/explain-printer
+                     with-out-str)]
+       {:path epath
+        :text estr
+        :id   epath}))
+   (::s/problems spec-ed)))
```

Comment:
This is what I have for each map - the `:path` and `:text` are self-explanatory, but for now I'm setting `:id` to be the path value. I'm not sure if it's worth investing time making a better ID value or if it should just be removed from the UI.

Reply:
that makes sense.

Reply:
Ok I decided to update the `:id` value to be of the form `specname-idx` (e.g. `profiles-0`) in https://github.com/yetanalytics/datasim/pull/65/commits/a2720dd580f431474eb69797f09fb7a4d5ff66ab. It seems like the cleanest solution to me.

BTW how do I find the value of the ID in developer tools?